token-lending/program/src/math/rate.rs

```diff
@@ -193,6 +193,6 @@ mod test {
 
     #[test]
     fn checked_pow() {
-        assert_eq!(Rate::one(), Rate::one().try_pow(u64::MAX).unwrap());
+        assert_eq!(Rate::one(), Rate::one().try_pow(u64::max_value()).unwrap());
```

Comment:
Got it. I read that it would be deprecated eventually, but the compiler was complaining about `u64::MAX`, so I wasn't sure whether to import it from std. I can replace these.

Reply:
Ah I see, where were you seeing that compiler error? If you update to solana tools v1.6.4 it should use rust 1.50.0 which would have that constant